mender-update/standalone.hpp

```diff
@@ -67,6 +67,14 @@ struct StateData {
 };
 using ExpectedOptionalStateData = expected::expected<optional::optional<StateData>, error::Error>;
 
+// Standalone script states:
```

Comment:
I dunno, do we want this somewhere. 

It felt useful when writing this, but now...